extension/package.json

```diff
@@ -775,7 +780,11 @@
         },
         {
           "command": "dvc.startExperimentsQueue",
-          "when": "dvc.commands.available && dvc.project.available && !dvc.experiment.running"
+          "when": "dvc.commands.available && dvc.project.available"
+        },
+        {
+          "command": "dvc.stopExperimentsQueue",
+          "when": "dvc.commands.available && dvc.project.available"
```

Comment:
The stop queue command is always available even when the queue isn't running. Would we want the command to only be available when the queue is running?

Reply:
We would need to interrogate `dvc queue status` to know whether or not there is anything running as the information is not readily available from `exp show` (queue can be started outside of VS Code, e.g in a terminal). It is essentially a no-op when the queue is not running. For now this is good enough.